Xamarin.Essentials/Background/Background.shared.cs

```diff
@@ -13,11 +13,7 @@ public static partial class Background
         public static void Add<T>(Func<T> schedule)
             where T : IBackgroundTask
         {
-#if NETSTANDARD1_0
```

Comment:
@webwarrior-ws why are you changing this file? this is only about Clipboard functionality, not BackgroundTasks